packages/itwin/breakdown-trees/.mocharc.jsonc

```diff
@@ -14,10 +14,10 @@
   ],
   "timeout": "60000",
   "file": [
-    "lib/test/setup.js"
+    "lib/cjs/test/setup.js"
   ],
   "ignore": [
-    "lib/test/coverage/**/*"
+    "lib/cjs/test/coverage/**/*"
```

Comment:
I'm not sure about this change. I thought coverage gets reported in the lib dir not lib/cjs. We def don't want to change it to the latter either because of the CI

Reply:
Reverted the change